.github/workflows/my-ci-pipeline.yml

```diff
@@ -41,3 +41,32 @@ jobs:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
           command: monitor
+
+  deployment:
+    name: Deploy to production
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - run: echo Publishing!
```

Comment:
I always find these lines unhelpful after the first ever run of them - worth just removing this step at this point?